kuma/static/js/analytics.js

```diff
@@ -18,6 +18,36 @@
                 win.ga('set', data.dimension, data.value);
             }
         },
+        /**
+         * Sends a statistic to GA dependant on the status of the win.ga object
+         * @param {object} data - The data to send to GA
+         * @param {function} [callback] - The function to call after sending data
+         */
+        send: function(data, callback) {
```

Comment:
I had lots of comments about this code, and then I saw that you just moved it, and the comment about Ghostery, and it all came back to me. Web is hard,